### PR TITLE
FISH-5824 Added Subresource Integrity for monitoring console

### PR DESCRIPTION
**Pull request by kalinchan**
## Description

Subresource Integrity (SRI) is a security feature that enables browsers to verify that resources they fetch (for example, from a CDN) are delivered without unexpected manipulation. It works by allowing you to provide a cryptographic hash that a fetched resource must match.
### Blockers

Dependent on https://github.com/payara/monitoring-console/pull/37
Will release version 1.8 of monitoring-console-webapp once approved.
### New tests

N/A
### Testing Performed

Loaded the monitoring console and verified everything works.
### Testing Environment

Ubuntu 20.04 openjdk version "1.8.0_302" maven 3.6.3
## Documentation

N/A

### CODE REVIEW

**Review by aubi — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -222,7 +222,7 @@
         <asm.version>9.1</asm.version>
         <monitoring-console-api.version>1.2</monitoring-console-api.version>
         <monitoring-console-process.version>1.7</monitoring-console-process.version>
```
Comment (Pandrex247) [inline pom.xml:224]: Despite the property, we're not in the habit of individually versioning the process vs. the webapp.
```suggestion
        <monitoring-console-process.version>1.8</monitoring-console-process.version>
```

**Comment by kalinchan** [comment]
jenkins test

**Comment by kalinchan** [comment]
jenkins test

**Pull request merged**